extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -117,27 +151,29 @@ public boolean isAnyPropertySet() {
     public static class HibernateOrmConfigDatabase {
 
         /**
-         * Control how schema generation is happening in Hibernate ORM.
-         * <p>
-         * Same as JPA's javax.persistence.schema-generation.database.action.
+         * Select whether the database schema is generated or not.
+         *
+         * `drop-and-create` is awesome in development mode.
+         *
+         * Accepted values: `none`, `create`, `drop-and-create`, `drop`, `update`. The default is `none`.
          */
         @ConfigItem
```

Comment:
Maybe add a `defaultValue="none"` to the annotation so that it is nicely added to the summary table :-)

Reply:
Yes, I have to tweak things a bit regarding that. And wondering if we could use the enum directly now but I have to test it. It's on my list of things to improve.

Reply:
An enum would be even better.

Reply:
Yup, I'm just not sure it will fly, I need to check it and wanted this PR out of the door.

I will fix the default value though. Small improvements are better than later improvements :).

Reply:
Done for the default value.